.github/workflows/update_tf.yml

```diff
@@ -48,9 +50,11 @@ jobs:
           body: |
             Updates TF to
             [${{ env.TF_SHA }}](https://github.com/tensorflow/tensorflow/commit/${{ env.TF_SHA }})
+            matching
+            [${{ env.TF_SHA }}](https://github.com/llvm/llvm-project/commit/${{ env.TF_SHA }})
             and copies over the LLVM BUILD files.
 
-            `./scripts/git/update_tf_llvm_submodules.py --llvm_commit=KEEP --update_build_files=true`
+            `./scripts/git/update_to_llvm_syncpoint.py`
```

Comment:
I've been wondering about this, but can we drop the newlines around the SHA links?

This ends up looking like
```
Updates TF to
[SHA]
matching
[SHA]
and copies over...
```

The newlines help when reading this file, but we should optimize for reading the commit messages. I'd prefer
```
Updates TF to [SHA] matching [SHA] and copies over...
```

Reply:
It's optimizing for reading the commit message in places that don't render the markdown (e.g. the command line). Same reason it's got explicit links even though GitHub will autolinkify. Just as markdown line wraps based on the source.

Reply:
Ah, fair point. I just use tools that render markdown or `git log --oneline`

Reply:
Yeah the real issue here is that GitHub does inconsistent markdown rendering in issues and PRs. This should be prettified by its markdown renderer :-/